doc/source/serve/advanced-guides/advanced-autoscaling.md

```diff
@@ -12,23 +12,26 @@ In this section, we go into more detail about Serve autoscaling concepts as well
+It is always recommended to load test your workloads. For example, if the use case is latency sensitive, you can lower the `target_ongoing_requests` number to maintain high performance. We recommend you benchmark your application code and set this number based on an end-to-end latency objective.
 
 :::{note}
-As an example, suppose you have two replicas of a synchronous deployment that has 100ms latency, serving a traffic load of 30 QPS. Then requests are assigned to replicas faster than the replicas can finish processing them; more and more requests are queued up at the replica (these are considered "ongoing requests") as time goes on, and then the average number of ongoing requests at each replica steadily increases. Latency will also increase since new requests have to wait for old requests to finish processing. If you set `target_num_ongoing_requests_per_replica = 1`, Serve detects a higher than desired number of ongoing requests per replica, and adds more replicas. At 3 replicas, your system would be able to process 30 QPS with 1 ongoing request per replica on average.
+As an example, suppose you have two replicas of a synchronous deployment that has 100ms latency, serving a traffic load of 30 QPS. Then requests are assigned to replicas faster than the replicas can finish processing them; more and more requests are queued up at the replica (these are considered "ongoing requests") as time goes on, and then the average number of ongoing requests at each replica steadily increases. Latency will also increase since new requests have to wait for old requests to finish processing. If you set `target_ongoing_requests = 1`, Serve detects a higher than desired number of ongoing requests per replica, and adds more replicas. At 3 replicas, your system would be able to process 30 QPS with 1 ongoing request per replica on average.
 :::
 
 #### **max_concurrent_queries [default=100] (DEPRECATED)**
 This has been renamed to `max_ongoing_requests`. `max_concurrent_queries` will be removed in a future release.
 
 #### **max_ongoing_requests [default=100]**
-There is also a maximum queue limit that is proxies respect when assigning requests to replicas. The limit is defined by `max_ongoing_requests`. We recommend setting `max_ongoing_requests` to ~20 to 50% higher than `target_num_ongoing_requests_per_replica`. Note that `target_num_ongoing_requests_per_replica` should always be strictly less than `max_ongoing_requests`, otherwise the deployment never scales up.
+There is also a maximum queue limit that is proxies respect when assigning requests to replicas. The limit is defined by `max_ongoing_requests`. We recommend setting `max_ongoing_requests` to ~20 to 50% higher than `target_ongoing_requests`. Note that `target_ongoing_requests` should always be strictly less than `max_ongoing_requests`, otherwise the deployment never scales up.
```

Comment:
```suggestion
There is also a maximum queue limit that proxies respect when assigning requests to replicas. Define the limit with `max_ongoing_requests`. Set `max_ongoing_requests` to ~20 to 50% higher than `target_ongoing_requests`. Note that `target_ongoing_requests` should always be strictly less than `max_ongoing_requests`, otherwise the deployment never scales up.
```